docs(getting-started): improve usage of `pnpm` (#9775)
nuxt/framework: docs/content/1.docs/1.getting-started/2.installation.md

```

```bash [pnpm]

> pnpm install --shamefully-hoist # or shamefully-hoist=true in .npmrc

Reviewer: ```suggestion
# Make sure you have `shamefully-hoist=true` in `.npmrc` before running pnpm install
pnpm install
```

> pnpm install #Notice the note below

Reviewer 2: I'd leave the info that was here before though. ☺️ 
```suggestion
pnpm install --shamefully-hoist # or shamefully-hoist=true in .npmrc
```

```

::

::alert

> **Note:** If using **pnpm**, make sure to have `.npmrc` with `shamefully-hoist=true` inside it before `pnpm install`. Or add `--shamefully-hoist` directly when `pnpm i`.

Reviewer 2: ```suggestion
**Note:** If using **pnpm**, make sure to have `.npmrc` with `shamefully-hoist=true` inside it before `pnpm install`. Otherwise, you have to add `--shamefully-hoist` when running `pnpm i`.
```

Author: Thank you for your better English!~ But I have some additional thoughts here.

Would  _Otherwise, you have to add `--shamefully-hoist` when running `pnpm i`._  be better?

Because it may feel easier to add `--shamefully-hoist` when running `pnpm i` than `.npmrc` with `shamefully-hoist=true` from the text before and after this part of the documentation.

Maybe _If using **pnpm**, you have to add `--shamefully-hoist` when running `pnpm i`. Otherwise, make sure to have `.npmrc` with `shamefully-hoist=true` inside it before `pnpm install`._ is more better?

Reviewer 3: > Because it may feel easier to add `--shamefully-hoist` when running `pnpm i` than `.npmrc` with `shamefully-hoist=true` from the text before and after this part of the documentation.

I disagree. Having `shamefully-hoist=true` in `.npmrc` is a lots better than having to include `--shamefully-hoist` every time you run `pnpm i`.

Reviewer 2: I'd also prefer a `.npmrc` as the "preferred" method because it is less error prone (forgetting the parameter / consistency throughout platforms etc. etc.).

Author: You are right! Now I deeply agree with both of you.

> **Note:** If using **pnpm**, make sure to have `.npmrc` with `shamefully-hoist=true` inside it before `pnpm install`. Otherwise, you have to add `--shamefully-hoist` when running `pnpm i`.

Reviewer: Since usage of flag won't remain on fresh lock files, and also makes detection harder, i think docs shall remain mentioning npmrc method only.

Author: I think you have a point. 

Also, I was wondering whether or not to introduce the thing that `pnpm dlx nuxi init` comes with `.npmrc` with `shamefully-hoist=true` compared to `npx nuxi init`
